models.py

```diff
@@ -1,14 +1,21 @@
 import datetime
 
 from flask import redirect, url_for, request
+from flask_admin.babel import lazy_gettext
+from flask_admin.contrib.sqla import tools
```

Comment:
not used

Reply:
removed tis line:
from flask_admin.contrib.sqla import tools